src/transformStateWithClones.js

```diff
@@ -7,7 +7,26 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newStates = []; // создаем пустой массив
+  let currentState = { ...state };
```

Comment:
[NAMING] - The variable name for the object copy should be more descriptive. Consider renaming `currentState` to `stateCopy` to better reflect its purpose.